concordia/static/css/base.css

```diff
@@ -357,6 +357,17 @@ ul.nav-secondary {
     opacity: 0.3 !important;
 }
 
+.btn-primary-alternative {
```

Comment:
I'm wondering whether we should explain anything more about the intended use of this class. Also, do we need to worry about having the same `:focus`, `:disabled`, and `:active` classes?

https://github.com/twbs/bootstrap/blob/dfba588d0d1ea020634f1a8f2911c7222702b125/scss/_buttons.scss#L7-L45

This is definitely a good argument for adopting Bootstrap SCSS where we could set the base theme colors and then make this just a modifier class.